release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -131,6 +131,12 @@ The following descheduler metrics are available:
 
 {product-title} 4.8 introduces the following notable technical changes.
 
+[discrete]
+[id="ocp-4-8-moderate-profiles"]
+==== The Federal Risk and Authorization Management Program (FedRAMP) moderate controls
+
+In {product-title} 4.8, the `rhcos4-moderate` profile is not complete. The `ocp4-moderate` profile will be completed in an upcoming release.
```

Comment:
"not complete" is a typo I did in the CMP jira. I'm sorry, I mean to say "is now complete".

To hopefully make it clear since I muddied the waters already so much (sorry!):

 - rhcos4-moderate is done and can be used in 4.8
 - ocp4-moderate is something we are working on now and hope to finish in time for 4.9 but I think we shouldn't make commitments to a particular release.

Reply:
@jhadvig Thanks for clarifying! Updated.